### ISSUE TITLE
hello. the data augment was not use in dataset?

### ISSUE THREAD

**Issue by aflyingnoob** [mpc001/end-to-end-lipreading#10]
I was suffer from overfitting when training the  video only model, and the cvtransfomer.py is not used in dataset? Thanks

**Comment by mpc001** [comment]
Hi, the data argumentation including horizontal flipping(probability=0.5) and random cropping are applied in main.py

**Comment by aflyingnoob** [comment]
Thanks, and sorry to disturb you about another question, my friends with me  have also trained your model and can not reach 83 with respect to the video only model. we are curious about if the npy data we use are wrong , could you please send  only one npy file to our for comparing (for example: one npy file like /lrw/about/test/ABOUT_00001.npy). my email is ```xing.zhao@vipl.ict.ac.cn``` .  Thank you very much.

**Comment by mpc001** [comment]
Hi,
Thanks for your interests. I have sent you. Please note that the coordinates we used. The coordinates for cropping in python is (FxHxW) = [:, 115:211, 79:175]

**Comment by aflyingnoob** [comment]
oh my god QAQ, which tool do you used in extracting the npy from mp4?matlab or ffmpeg? If ffmpeg, is it convinent to provide version and arguments? Thanks

**Comment by mpc001** [comment]
Hi,
I used Matlab (mmread) for extracting. The reason for saving .npy format is for speed. Both mmread and opencv-python have options to specify ffmpeg as backend. If you would like to directly extract from mp4, you could check the previous issue #6.

**Comment by aflyingnoob** [comment]
我的天啊我试了好多库... 视频转成帧后都和您发给我的那个npy差几个值, 放弃了....谢谢您的回答.

**Comment by aflyingnoob** [comment]
您好,方便提供下训练/验证/测试分别有多少条视频吗?谢谢

**Comment by mpc001** [comment]
Hi, @aflyingnoob,

Thanks for your interests.

I used opencv-python for reading video in python2.7 environment. The validation accuracy is 0.8358 while testing accuracy is 0.8329.

In total there are 488766, 25000, and 25000 examples in the training, validation and test sets.

**Comment by wanli4717** [comment]
Hello, I am also reproducing the experimental results of this paper according to your code recently. There may be some problems in the NPY generated by my experiment, which can not meet the accuracy of the current experiment. Could you send me a python file (wanli@tju.edu.cn) that you generated for npy? Thank you very much.

**Comment by mpc001** [comment]
Hi, Wanli,

Thanks for your interests!

In the experiment, we used mmread on matlab for reading raw videos. That's what we did in the paper. Please note that the fixed mouth region in LRW (FxHxW) = [:, 115:211, 79:175] in the python environment.
